README.md

```diff
@@ -116,9 +116,9 @@ Thomas' values were heavily influenced by his experience growing up playing spor
 - Friendly - Thomas was voted "friendliest" throughout junior and high school. 
 - Courteous - Thomas strives to be as polite as possible. 
 - Kind - Thomas considers kindness to be a virtue and strives to be as kind as possible. 
-- Obedient
-- Cheerful - Thomas Strives to be a cheerful individual.
-- Thrifty
-- Brave
-- Clean
+- Obedient - Thomas is conscientious about following policies he regularly interacts with, e.g. rules around sox compliance.
```

Comment:
I thought these entries without detail looked a bit blank/unfinished and filled them in, again I'm making assumptions based on our limited interactions and how I think you might apply the Scout Law to your work and life.